src/test/java/org/jenkinsci/plugins/gitclient/WorkspaceWithRepo.java

```diff
@@ -92,7 +93,8 @@ String localMirror() throws IOException, InterruptedException {
                      * the final destination directory.
                      */
                     Path tempClonePath = Files.createTempDirectory(targetDir.toPath(), "clone-");
-                    cliGitCommand.run("clone", "--reference", f.getCanonicalPath(), "--mirror", repoURL, tempClonePath.toFile().getAbsolutePath());
+                    cliGitCommand.run("clone", "--reference", f.getCanonicalPath(), "--mirror", repoURL,
+                            tempClonePath.toFile().getAbsolutePath());
```

Comment:
Please remove the pure white space changes in the existing files.
```suggestion
                    cliGitCommand.run("clone", "--reference", f.getCanonicalPath(), "--mirror", repoURL, tempClonePath.toFile().getAbsolutePath());
```